xrayvision/tests/test_visibility.py

```diff
@@ -70,3 +84,15 @@ def test_vis_phase_uncertainty(visibilities):
     expected_phase_uncertainty = [0.22918312, 1.4323945] * apu.deg
 
     assert_quantity_allclose(output_phase_uncertainty, expected_phase_uncertainty)
+
+
+def test_vis_eq(visibilities):
+    vis = visibilities
+    assert vis == vis
+
+
+def test_meta_eq(vis_meta):
+    meta = vis_meta
+    assert meta == meta
```

Comment:
Wouldn't this always be the case a meta is defined to be meta?

Reply:
Fair point.  But in order for this to pass, the metas have to be the same.  The value of this test is that if `__eq__` is broken, this test won't pass.